xml/System.Runtime.CompilerServices/RuntimeHelpers.xml

```diff
@@ -1129,8 +1129,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>This method ensures that there is sufficient stack to execute the average Framework function. If there is not enough stack, then it returns false.</summary>
+        <returns><see langword="true" /> if sufficient stack is present; otherwise, <see langword="false" />.</returns>
         <remarks>To be added.</remarks>
```

Comment:
I would also add the remark section and the note from `EnsureSufficientExecutionStack` (if applies).
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.runtimehelpers.ensuresufficientexecutionstack?view=netcore-3.1#remarks

Reply:
Does this remark apply, @Anipik or did you decide it was not necessary?

Reply:
I think its not necessary here. There was nothing extra information that needs to be added in remarks of TryEnsureSufficientExecutionStack which is not already there.